src/components/App/Show/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchShow } from 'redux/actions/showsActions';
+
+import styles from './Show.module.scss';
+
+class Show extends Component {
+  componentDidMount() {
+    const { id } = this.props.match.params;
+    this.props.fetchShow(id);
+  }
+
+  render() {
+    const { name, image, summary, runtime, status, type, schedule } = this.props.show;
+    const { time } = schedule || '';
+    const days  = schedule ? schedule.days.join(', ') : '';
+
+    if (!image) {
+      return <p>Loading...</p>;
+    }
+
+    return (
+      <>
+        <figure className={styles.showInfo}>
+          <img src={image.original} alt={name} className={styles.showImage}/>
+            <figcaption className={styles.description}>
+              <h3>{name}</h3>
+              <p className={styles.summary}>{summary.replace(/(<([^>]+)>)/ig,"")}</p>
```

Comment:
I saw this replace in other place - maybe use one function instead duplicate code?

Reply:
It wasn't about React Fragments, it's about summary.replace function which you're using in two places. Maybe it makes sense to add this to helpers

Reply:
Ah, right. I agree, it should be moved to a helper, I'm introducing the change.